### PR TITLE
Bug 1757573, updated Cluster Loader docs

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1757573

Pretty preview build: http://file.rdu.redhat.com/~ahardin/10112019/4-2-cluster-loader/scalability_and_performance/using-cluster-loader.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sjug Thanks for your guidance on these updates! Please review and let me know if I missed anything.

**Comment by ahardin-rh** [comment]
@mffiedler PTAL as well. Thanks!

**Comment by mffiedler** [comment]
@wabouhamad Please review

**Comment by wabouhamad** [comment]
@ahardin-rh sorry took me a while to build a cluster to test the commands, I am still unable to run the sample commands successfully

File: modules/running-cluster-loader.adoc [hunk 1]
```diff
@@ -11,8 +11,8 @@
 template builds and waits for them to complete:
 +
 ----
-$ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config -i
-quay.io/openshift/origin-tests:4.1 /bin/bash -c 'export KUBECONFIG=/root/.kube/config && \
+$ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config:z -i
```
Comment (wabouhamad) [inline modules/running-cluster-loader.adoc]: we need a backslack for continuation of command at the of this line

File: modules/running-cluster-loader.adoc [hunk 2]
```diff
@@ -21,14 +21,17 @@ Alternatively, execute Cluster Loader with a user-defined configuration by
 setting the environment variable for `VIPERCONFIG`:
 +
 ----
-$ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config -i
-quay.io/openshift/origin-tests:4.1 /bin/bash -c 'export KUBECONFIG=/root/.kube/config && \
-export VIPERCONFIG=config/test && \
-openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the \
-cluster [Suite:openshift]"'
+$ sudo podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config:z
```
Comment (wabouhamad) [inline modules/running-cluster-loader.adoc]: Also here need a backslack for continuation of command at the of this line

**Review by wabouhamad — CHANGES_REQUESTED** [review]
When I run these commands on a jump host, I get the following error `podman run -v ${LOCAL_KUBECONFIG}:/root/.kube/config:z -i quay.io/openshift/origin-tests:4.2 /bin/bash -c 'export KUBECONFIG=/root/.kube/config && \
openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the \
cluster [Suite:openshift]"'
I1014 18:21:20.283831       8 test.go:182] openshift-tests excluded test regex is "\\[Disabled:|\\[Disruptive\\]|\\[Skipped\\]|\\[Slow\\]|\\[Flaky\\]|\\[local\\]|\\[Suite:openshift/test-cmd\\]|\\[Skipped:skeleton\\]|\\[Skipped:Network/OpenShiftSDN\\]"
Oct 14 18:21:20.369: INFO: >>> kubeConfig: /root/.kube/config
Oct 14 18:21:20.370: INFO: Waiting up to 30m0s for all (but 100) nodes to be schedulable
Oct 14 18:21:20.393: INFO: Waiting up to 10m0s for all pods (need at least 0) in namespace 'kube-system' to be running and ready
Oct 14 18:21:20.416: INFO: 0 / 0 pods in namespace 'kube-system' are running and ready (0 seconds elapsed)
Oct 14 18:21:20.416: INFO: expected 0 pod replicas in namespace 'kube-system', 0 are Running and Ready.
Oct 14 18:21:20.416: INFO: Waiting up to 5m0s for all daemonsets in namespace 'kube-system' to start
Oct 14 18:21:20.424: INFO: e2e test version: v1.14.6+f9b5405
Oct 14 18:21:20.425: INFO: kube-apiserver version: v1.14.6+2e5ed54
[BeforeEach] [Top Level]
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/test.go:59
[BeforeEach] [Feature:Performance][Serial][Slow] Load cluster
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/framework.go:149
STEP: Creating a kubernetes client
Oct 14 18:21:20.428: INFO: >>> kubeConfig: /root/.kube/config
[BeforeEach] [Feature:Performance][Serial][Slow] Load cluster
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cluster/cl.go:44
Oct 14 18:21:20.431: INFO: Undefined config file, using built-in config /tmp/fixture-testdata-dir668249003/test/extended/testdata/cluster/master-vert.yaml

[It] should load the cluster [Suite:openshift]
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/cluster/cl.go:65
Oct 14 18:21:20.432: INFO: Our tuning set is: &{default {{5 0s 0s} {0s}} {{0 0s 0s} {0s}}}
Oct 14 18:21:20.440: INFO: Configuration requested deletion of project clusterloader-cakephp-mysql0
Oct 14 18:21:20.440: INFO: Deleting project clusterloader-cakephp-mysql0 ...
Oct 14 18:21:30.460: INFO: Running 'oc --namespace= --config= adm new-project clusterloader-cakephp-mysql0'
Created project clusterloader-cakephp-mysql0
Oct 14 18:21:30.572: INFO: 1/1 : Created new namespace: clusterloader-cakephp-mysql0
[AfterEach] [Feature:Performance][Serial][Slow] Load cluster
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/util/client.go:120
[AfterEach] [Feature:Performance][Serial][Slow] Load cluster
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/framework.go:150
Oct 14 18:21:30.573: INFO: Waiting up to 3m0s for all (but 100) nodes to be ready
Oct 14 18:21:30.582: INFO: Running AfterSuite actions on all nodes
Oct 14 18:21:30.582: INFO: Running AfterSuite actions on node 1
fail [github.com/openshift/origin/test/extended/cluster/cl.go:156]: Unexpected error:
    <*errors.errorString | 0xc003d59e20>: {
        s: "unable to find pod/template file ./test/extended/testdata/cluster/quickstarts/cakephp-mysql.json\n",
    }
    unable to find pod/template file ./test/extended/testdata/cluster/quickstarts/cakephp-mysql.json
    
occurred` not sure if something else needs to be pre-configured

**Comment by sjug** [comment]
@wabouhamad That's a bug with Cluster Loader, not with the docs.

**Comment by wabouhamad** [comment]
@sjug should we mention that somewhere, and list Bug id, someone may try these commands and hit that failure ?  If CL bug is being fixed before 4.2 GA, and commands are correct, that would probably be OK

**Comment by ahardin-rh** [comment]
@sjug @wabouhamad Yeah, if there is a known issue that will stand for 4.2, we should get that noted here. Do you have a BZ number?

File: modules/configuring-cluster-loader.adoc [hunk 3]
```diff
@@ -5,11 +5,8 @@
 [id="configuring_cluster_loader_{context}"]
 = Configuring Cluster Loader
 
-The tool creates multiple namespaces (projects), which contain multiple templates or pods.
-
-Locate the configuration files for Cluster Loader in the `config/` subdirectory.
-The pod files and template files referenced in these configuration examples are
-found in the `content/` subdirectory.
+The tool creates multiple namespaces (projects), which contain multiple
+templates or pods.
```
Comment (bergerhoffer) [inline modules/configuring-cluster-loader.adoc]: Pods

File: modules/running-cluster-loader.adoc [hunk 4]
```diff
@@ -21,14 +21,17 @@ Alternatively, execute Cluster Loader with a user-defined configuration by
...
+/bin/bash -c 'KUBECONFIG=/root/.kube/config VIPERCONFIG=/root/configs/test.yaml \
+openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should \
+load the cluster [Suite:openshift]"'
 ----
 +
-In this example, there is a subdirectory called *_config/_* with a configuration
-file called *_test.yml_*. In the command line, exclude the extension of the
-configuration file, as the tool will automatically determine the file type and
-extension.
+In this example, `${LOCAL_KUBECONFIG}` refers to the path to the `kubconfig` on
```
Comment (bergerhoffer) [inline modules/running-cluster-loader.adoc]: kubeconfig

**Comment by wabouhamad** [comment]
@ahardin-rh LGTM

**Pull request merged**